{{cookiecutter.repo_name}}/setup.py

```diff
@@ -40,20 +40,33 @@
     # Customize MANIFEST.in if the general case does not suit your needs
     # Comment out this line to prevent the files from being packaged with your software
     include_package_data=True,
-
+    python_requires=">=3.8",          # Python version restrictions
     # Allows `setup.py test` to work correctly with pytest
     setup_requires=[] + pytest_runner,
-
+    # Required packages, pulls from pip if needed
+    # do not use for Conda deployment
+    install_requires=[
+        "mdanalysis",
```

Comment:
This should match the pyproject.toml
```suggestion
        "mdanalysis>=2.0.0",
```